server/setup.py

```diff
@@ -13,5 +13,8 @@
             '../libexec/pulp-manage-workers = pulp.server.async.manage_workers:main',
             'pulp-manage-db = pulp.server.db.manage:main',
         ]
-    }
+    },
+    install_requires=[
+        'celery >=3.1.0, <3.2.0', 'httplib2', 'iniparse', 'isodate>=0.5.0', 'm2crypto',
+        'mongoengine>=0.7.10', 'oauth2>=1.5.211', 'pymongo>=2.5.2', 'setuptools', 'web.py']
```

Comment:
On 02/23/2015 10:39 AM, bmbouter wrote:
> @rbarlow <https://github.com/rbarlow> How is the inclusion of Django
> here related to this bug?
> 
> https://bugzilla.redhat.com/show_bug.cgi?id=1195254

Adding Django here will cause Django to get installed via pip by the
dev-setup.sh script, but once we fix that bug it will get installed via
yum instead.

We should aim for nothing but the repos themselves getting installed by pip.